data/inline-histogram.go

```diff
@@ -178,6 +179,20 @@ func (ih *InlineHistogram) String() string {
 	}
 	return s
 }
+
+func (ih *InlineHistogram) HistogramToEmpiricalString() string {
```

Comment:
it seems like you can produce an Empirical distribution from bins and bincounts - why not just create an exported function to access that protected memory in the struct?